go/base/static/js/src/components/plumbing/endpoints.js

```diff
@@ -35,13 +35,13 @@
     // connection source/target
     plumbSourceOptions: function() {
       return {
-        anchor: 'Continuous',
+        anchor:['Continuous', { faces:[ 'top', 'bottom' ] } ],
         filter: this.dragFilter
       };
     },
 
     plumbTargetOptions: {
-      anchor: 'Continuous',
+      anchor:['Continuous', { faces:[ 'top', 'bottom' ] } ],
```

Comment:
Sorry, bikeshed, but could we keep with our usual conventions here: `anchor: ['Continuous', {faces:[ 'top', 'bottom']}]`